tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -172,8 +173,20 @@ export class ScalarCardComponent<Downloader> {
           ...datum.metadata,
           closest: false,
           distSqToCursor: Math.hypot(
-            datum.point.x - cursorLoc.x,
-            datum.point.y - cursorLoc.y
+            datum.point.x - cursorLocationInDataCoord.x,
+            datum.point.y - cursorLocationInDataCoord.y
+          ),
+          distSqToCursorPixels: Math.hypot(
```

Comment:
In this calculation datum.point.x is data space while cursorLocation.x is pixel space, right? For the calculation to make sense they'd both have to be in pixel space.

Reply:
Great catch, I routed the pixel locations through and used those instead